scripts/install-windows-prereqs.ps1

```diff
@@ -40,7 +40,7 @@ Param(
 $ErrorActionPreference = "Stop"
 
 $PACKAGES_DIRECTORY = Join-Path $env:TEMP "packages"
-$OE_NUGET_DIR = Join-Path $InstallPath "prereqs\nuget"
+$OE_NUGET_DIR = $InstallPath
```

Comment:
If you are changing this, consider changing to use some default nuget package folder that can be documented consistently across the CMakeSettings.json file used with VS2017 (e.g. Can we reuse the nuget global-packages path by capturing it to an environment variable, or saving this one, for use in the CMakeSettings.json?)

Reply:
I'll create an issue for refactoring this. It would be nice if we at least support pulling NuGet packages from a user's default NuGet package install directory, but also allow people to override it.